Assets/Scripts/UI/PickupPup/DogWorld/DogWorldSlot.cs

```diff
@@ -50,5 +50,15 @@ void handleScoutingDone()
     {
         this.Show();
     }
-        
+
+    /* BP Unsure if this is the preferred spot for this, but we apparently
+     * don't want any click noises for the dogs in the scene (yard/living room) */
+    void Start()
+    {
+        UISFXHandler sfxScript = GetComponent<UISFXHandler>();
+        sfxScript.clickEnabledSoundEvent = "";
```

Comment:
Done. That's a common thing I've done in the past so thanks for pointing that out. Much cleaner, indeed.